### DISCUSSION TITLE
Paddle ocr structure 和 ocr model 在 H800 GPU 上输出结果为 空

### DISCUSSION THREAD

**Discussion by itsliupeng** [PaddlePaddle/PaddleOCR#12661] [Q&A]
- 系统环境/System Environment： GPU H800, NVIDIA-SMI 525.105.17   Driver Version: 525.105.17   CUDA Version: 12.1
- 版本号/Version：Paddle： paddlepaddle-gpu    2.5.1.post120 

运行命令：paddleocr --image_dir=ppstructure/docs/table/1.png --type=structure --table=false --ocr=true

使用 https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/ppstructure/README_ch.md 这里面的模型来识别 pdf
Paddle ocr structure 和 ocr model 在 H800 GPU 上输出结果为 空， 但在 CPU 和 A800 GPU  上正常，请问下具体原因

**Comment by zhangjx123** [comment]
我也遇到了相同的问题，使用H800，模型初始化后的预测结果都不同，有时为空。同台机器上，不使用H800只使用CPU可正常输出

**Comment by SirlyDreamer** [comment]
我也遇到了相同的问题，使用H800模型预测结果为空，但是使用CPU预测正常

**Comment by TQC10** [comment]
我也遇到了在H800上使用GPU预测输出为空的问题，在使用CPU预测时可以正常输出

**Comment by SWHL** [comment]
这种感觉是PaddlePaddle推理框架的原因，上边遇到的小伙伴，可以验证一下paddlepaddle-gpu是否可以正常使用呢？
```python
import paddle

paddle.utils.run_check()
# 看看会不会说successfully
```

**Comment by xh-zheng** [comment]
26年了还没解决，H800跑PP-OCRv5结果为空，CPU正常
